spec/module.dd

```diff
@@ -177,8 +177,8 @@ import foo;  // Deprecated: module foo is deprecated - Please use foo2 instead.
     $(LI $(GLINK PackageName)s and $(GLINK ModuleName)s should be composed of the ASCII
     characters lower case letters, digits or `_` to ensure maximum portability and compatibility with
     various file systems.)
-    $(LI The file names for packages and modules should also be composed of
-    the ASCII characters lower case letters, digits or `_` and should not be a $(GLINK_LEX Keyword).)
+    $(LI The file names for packages and modules should be composed only of
```

Comment:
"must be composed of" ? We could use more RFC2119 in this text.

Reply:
I believe my version is clearer than the original. Quoting more RFC2119 is a separate matter.